src/Field.js

```diff
@@ -61,6 +61,7 @@ export default {
 
     const children = this.$scopedSlots.default({
       events: this.fieldEvents,
+      change,
```

Comment:
so it seems you can access `change` via `meta.change` currently while the doc says the opposite: 😅

<img width="526" alt="2018-12-19 10 37 41" src="https://user-images.githubusercontent.com/8784712/50226658-bec95780-03de-11e9-9f0b-8cf649f238b5.png">

This PR would fix this mistake 👌🏻

Can you please add `change` to `Scoped slot props` in README?

Reply:
I think we already have it in README